src/coldcard.js

```diff
@@ -29,33 +29,27 @@ import {
   INFO,
   ERROR,
 } from "./interaction";
-import {
-  P2SH,
-  P2SH_P2WSH,
-  P2WSH,
-} from 'unchained-bitcoin';
+import { P2SH, P2SH_P2WSH, P2WSH } from "unchained-bitcoin";
 
-export const COLDCARD = 'coldcard';
+export const COLDCARD = "coldcard";
 // Our constants use 'P2SH-P2WSH', their file uses 'P2SH_P2WSH' :\
 export const COLDCARD_BASE_BIP32_PATHS = {
   "m/45'": P2SH,
-  "m/48'/0'/0'/1'": P2SH_P2WSH.replace('-', '_'),
+  "m/48'/0'/0'/1'": P2SH_P2WSH.replace("-", "_"),
   "m/48'/0'/0'/2'": P2WSH,
-  "m/48'/1'/0'/1'": P2SH_P2WSH.replace('-', '_'),
+  "m/48'/1'/0'/1'": P2SH_P2WSH.replace("-", "_"),
   "m/48'/1'/0'/2'": P2WSH,
 };
 const COLDCARD_BASE_CHROOTS = Object.keys(COLDCARD_BASE_BIP32_PATHS);
 
-export const COLDCARD_WALLET_CONFIG_VERSION = "0.0.1";
-
+export const COLDCARD_WALLET_CONFIG_VERSION = "0.0.2";
```

Comment:
Does this version come from coldcard or is it our own internal tracking?

Reply:
internal tracking

Reply:
Got it. I like this idea in general, of versioning config standards. Depending on what direction we go in with #44 this might be something to consider tracking for all configs we support (including caravan's JSON). 

Wonder what you think of making this a bigger bump then a "patch"? I understand this is kind of arbitrary, but given that these two versions are basically not compatible, might be worth making it `0.1.0` or even `1.0.0`? I'm not super opinionated either way, but thought I'd throw it out there.

Reply:
changing it to `1.0.0`